### PR TITLE
Bug Fix: Calculate DVHs Messagebox button ordering 

### PR DESCRIPTION
**Pull request by LWMcClymont**
Calculate DVHs Messagebox now displays "No" & "Yes" buttons in respective order across platforms.

### CODE REVIEW

File: src/View/ProgressWindow.py [hunk 1]
```diff
@@ -59,15 +60,58 @@ def update_progress(self, progress_update):
...
+                respect to their assigned roles. (0 first, then 0 and so on) 'AcceptRole' is 0 and 'RejectRole' is 1 
+                thus by counterintuitively assigning 'No' to 'AcceptRole' and 'Yes' to 'RejectRole' the buttons are 
+                positioned as desired.
+            """
+            mb.addButton(button_no, QtWidgets.QMessageBox.AcceptRole)
+            mb.addButton(button_yes, QtWidgets.QMessageBox.RejectRole)
+
+            # Apply stylesheet to the message box and add icon to the window
+            mb.setStyleSheet(open(stylesheet_path).read())
+            mb.setWindowIcon(QtGui.QIcon(resource_path("res/images/btn-icons/onkodicom_icon.png")))
```
Comment (sjswerdloff) [inline src/View/ProgressWindow.py:107]: Is this expression of path platform independent?  If not, suggest using pathlib.

File: src/View/mainpage/DVHTab.py [hunk 2]
```diff
@@ -133,14 +133,60 @@ def plot_dvh(self):
...
+                       "minutes on some systems.)")
+            button_no = QtWidgets.QPushButton("No")
+            button_yes = QtWidgets.QPushButton("Yes")
+
+            """ We want the buttons 'No' & 'Yes' to be displayed in that exact order. QMessageBox displays buttons in
+                respect to their assigned roles. (0 first, then 0 and so on) 'AcceptRole' is 0 and 'RejectRole' is 1 
+                thus by counterintuitively assigning 'No' to 'AcceptRole' and 'Yes' to 'RejectRole' the buttons are 
+                positioned as desired.
+            """
+            mb.addButton(button_no, QtWidgets.QMessageBox.AcceptRole)
```
Comment (sjswerdloff) [inline src/View/mainpage/DVHTab.py:176]: That very similar code is being repeated suggests one might want to create a function that gets shared / reused.

File: src/View/ProgressWindow.py (hunk 1 above)
Comment (LWMcClymont) [reply src/View/ProgressWindow.py]: I believe it was, it's been updated to use pathlib regardless.

File: src/View/mainpage/DVHTab.py (hunk 2 above)
Comment (LWMcClymont) [reply src/View/mainpage/DVHTab.py:178]: Yes, you are correct, however, I could not find a simple clean way of creating a single function that wouldn't require passing references from class to class which would get a little messy.
I suspect this is why the code was written the way it originally was.
 
My initial solution was to make the function static from within DVHTab that the other objects can reference, however, QMessageBox requires a reference to an instance of the parenting classes QWidget class. This cannot be accessed if the function were static.
From my understanding, this is where the styling of the buttons and such comes from. 
I could create a new QMessageBox object and assign the stylesheet as I have done for the windows correction, however on Linux versions, the style differentiates from what is currently shown on master for an unknown reason. 
I did not want to change too much of what is currently working. Please if you have any suggestions they are warmly welcome.

Comment (sjswerdloff) [reply src/View/mainpage/DVHTab.py:178]: OK.  You examined the problem and there wasn't a reasonable solution that presented itself.  Fair enough!

File: src/View/ProgressWindow.py [hunk 3]
```diff
@@ -59,15 +61,59 @@ def update_progress(self, progress_update):
...
+                self.signal_advise_calc_dvh.emit(True)
+            else:
+                self.signal_advise_calc_dvh.emit(False)
         else:
-            self.signal_advise_calc_dvh.emit(False)
+            stylesheet_path = ""
+
+            # Select appropriate style sheet
+            if platform.system() == 'Darwin':
+                stylesheet_path = "res/stylesheet.qss"
```
Comment (sjswerdloff) [inline src/View/ProgressWindow.py:84]: Not using pathlib here

**Review by sjswerdloff — APPROVED** [review]
Consider following through with pathlib on some resource paths.

File: src/View/ProgressWindow.py (hunk 3 above)
Comment (LWMcClymont) [reply src/View/ProgressWindow.py]: Fixed - Thank you for spotting.

**Pull request merged**